data-prepper-plugins/common/src/test/java/com/amazon/dataprepper/plugins/sink/StdOutSinkTests.java

```diff
@@ -12,21 +13,47 @@
 package com.amazon.dataprepper.plugins.sink;
 
 import com.amazon.dataprepper.model.configuration.PluginSetting;
+import com.amazon.dataprepper.model.event.Event;
+import com.amazon.dataprepper.model.event.JacksonEvent;
 import com.amazon.dataprepper.model.record.Record;
 import org.junit.Test;
+import org.junit.jupiter.api.BeforeEach;
 
-import java.util.Arrays;
+import java.util.ArrayList;
 import java.util.HashMap;
 import java.util.List;
+import java.util.Map;
+import java.util.UUID;
 
 public class StdOutSinkTests {
     private static String PLUGIN_NAME = "stdout";
 
-    private final String TEST_DATA_1 = "data_prepper";
-    private final String TEST_DATA_2 = "stdout_sink";
-    private final Record<String> TEST_RECORD_1 = new Record<>(TEST_DATA_1);
-    private final Record<String> TEST_RECORD_2 = new Record<>(TEST_DATA_2);
-    private final List<Record<String>> TEST_RECORDS = Arrays.asList(TEST_RECORD_1, TEST_RECORD_2);
+    private final Map<String, Object> TEST_DATA_1 = new HashMap<>();
```

Comment:
I recommend that you make this a non-final and initialize it in the `setup`. As it is right now, authors will need to clear the map in-between tests.

The same goes for `TEST_DATA_2` and `TEST_RECORDS`.